content/en/docs/concepts/webhook.md

```diff
@@ -44,5 +44,4 @@ following two Secrets:
 If errors occur around the webhook but the webhook is running then the webhook
 is most likely not reachable from the API server. In this case, ensure that the
 API server can communicate with the webhook by following the [GKE private
-cluster explanation](/docs/installation/compatibility/). TODO: add back in
-'#GKE'.
+cluster explanation](/docs/installation/compatibility/#gke).
```

Comment:
We should avoid using absolute paths - it makes navigating when we create a versioned directory difficult as it'll send users to the 'latest' version of the docs.